syft/__init__.py

```diff
@@ -1 +1,2 @@
 from .tensor import TensorBase
+from .tensor import *
```

Comment:
I think it's not a good practice to have this kind of imports in Python. It is always better to be explicit about what you are importing.